hazelcast/src/main/java/com/hazelcast/datalink/impl/HazelcastDataLinkRegistration.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.datalink.impl;
+
+import com.hazelcast.datalink.DataLink;
+import com.hazelcast.datalink.DataLinkRegistration;
+import com.hazelcast.datalink.HazelcastDataLink;
+
+/**
+ * Registration for {@link HazelcastDataLink}
+ */
+public class HazelcastDataLinkRegistration implements DataLinkRegistration {
+
+    @Override
+    public String type() {
+        return "HZ";
```

Comment:
Shouldn't it be "IMap"? I know that usage of HazelcastDataLink in GenericMapStore doesn't make much sense, but I think we should stay consistent